### PR TITLE
Adding thatool_package

### PR DESCRIPTION
**Pull request by thangckt**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/thatool/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 843304b7768932e6231f3f5ed5f8946b3ec3ebd3d4ac4a75bfba7ea85604739f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
```
Comment (ngam) [inline recipes/thatool/meta.yaml:15]:
```suggestion
  number: 0
  noarch: python
  script: "{{ PYTHON }} -m pip install . -vv"
  
```

Let's try to make this a noarch.

File: recipes/thatool/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
```
Comment (ngam) [inline recipes/thatool/meta.yaml:23]: Add a pip check and activate the tests below pls

File: recipes/thatool/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    # - thatool.utils
+
+about:
+  home: The package home page
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: "This package contains the docs for several in-house codes to handle some specific tasks."
+  doc_url: 
+  dev_url:
```
Comment (ngam) [inline recipes/thatool/meta.yaml:39]: fill these fields pls (include home above)

**Review by ngam — CHANGES_REQUESTED** [review]

File: recipes/thatool/meta.yaml (hunk 2 above)
Comment (ngam) [reply recipes/thatool/meta.yaml:24]:
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```

File: recipes/thatool/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
```
Comment (ngam) [inline recipes/thatool/meta.yaml:22]:
```suggestion
    - python >=3.6 
  run:
    - python >=3.6
```

**Comment by thangckt** [comment]
done

File: recipes/thatool/meta.yaml (hunk 3 above)
Comment (thangckt) [reply recipes/thatool/meta.yaml:53]: done

Comment (dopplershift) [reply recipes/thatool/meta.yaml:53]: Can you either give `dev_url` a value or remove it?

**Review by dopplershift — APPROVED** [review]

**Pull request merged**